Sources/TuistGraph/DependenciesGraph/ThirdPartyDependency.swift

```diff
@@ -63,22 +63,76 @@ extension ThirdPartyDependency {
         /// The paths containing the target resources.
         public let resources: [AbsolutePath]
 
-        /// The target dependencies
+        /// The target dependencies.
         public let dependencies: [Dependency]
 
-        // TODO: check and add any other information needed to compile the sources (e.g. build flags)
+        /// The custom public headers path.
+        public let publicHeadersPath: String?
 
-        public init(name: String, sources: [AbsolutePath], resources: [AbsolutePath], dependencies: [Dependency]) {
+        /// The header search paths for C code.
+        public let cHeaderSearchPaths: [String]
+
+        /// The header search paths for CXX code.
```

Comment:
I think in comments we can write out `C++` instead of `CXX` (applicable to more comments here)

Reply:
✅